lib/Resource.js

```diff
@@ -88,6 +90,22 @@ var Resource = function(options) {
     };
 
   });
+
```

Comment:
can you please add a comment here to describe what this is doing, e.g. ```// cache association data for later use``` or something